DaoBuilder/dcntDAOConfig.ts

```diff
@@ -0,0 +1,26 @@
+import { ethers } from "ethers";
+import { DecentDAOConfig } from "./types";
+
+export const decentDAOConfig: DecentDAOConfig = {
+  lockStart: Math.floor(Date.now() / 1000), // (Now) Start time of token lock
+  lockDuration: 60 * 60 * 24 * 365, // 1 year
+  snapshotURL: "https://snapshot.org/#/decentdao.eth",
+  initialSupply: "1000000",
+  votingPeriod: 86400, // Length of time (in blocks) that voting occurs.
+  quorum: 4, // Percentage of total possible tokens that must vote in order to consider a proposal result valid.  We should take into account that a large portion of tokens will be locked for investors, who may never vote.
```

Comment:
quorum is out of 1 million, for greater precision, so 4% here would be `40000`

Reply:
So I put this as 40000 and it showed up in Fractal interface as 40000% lol.